tests/unit/moduleapi/cmdintrospection.tcl

```diff
@@ -37,6 +37,7 @@ start_server {tags {"modules"}} {
         dict unset redis_reply group
         dict unset module_reply group
         dict unset module_reply module
+        dict unset redis_reply reply_schema
```

Comment:
@guybe7 , don't really like the fact we add it to the test-suite, any other suggestions?

Reply:
please pay attention that the basic configuration test-suite will be failed, not only resp3 runs. We cannot merge the reply-schema branch in that way. Merge this PR, but will have to find a solution.